src/pages/Upload/components/UploadInfo/UploadPage/CategoryPage/SubmitPage.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import styled from 'styled-components';
+import ButtonContainer from './ButtonContainer';
+
+const SubmitPage = ({ children, page }) => {
+  const pageSelector = () => {
+    if (page === 0) {
+      return 'disable';
+    } else if (page === 1) {
+      return 'active';
+    }
+  };
+  console.log(page);
+  return <Submit className={pageSelector}>{children}</Submit>;
```

Comment:
- 여기도 props 이용해 보세요! 다음과 같이 말이죠!
```suggestion
  return <Submit isDisabled={page === 0 ? true : false}>{children}</Submit>;
  
  ...
  
const Submit = styled.div`
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  opacity: {props => props.isDisabled ? 0 : 1}
  
  ...
```

Reply:
전환 완료했습니다!!